docker_entrypoint.sh

```diff
@@ -1,3 +1,26 @@
 #!/bin/sh
 
-exec tini hello-world
+set -e
+
+ITCHYSATS_USER=itchysats
+ITCHYSATS_PASSWORD=$(yq e '.password' /root/start9/config.yaml)
+
+# Properties Page
+echo 'version: 2' > /root/start9/stats.yaml
```

Comment:
Might be better to do a multiline echo, e.g. 

echo "version: 2
data: 
  ItchySats Username: 
...
" >> /root/start9/stats.yaml
```